app/scripts/app.js

```diff
@@ -271,6 +271,16 @@ angular
         controller: 'ServiceController',
         reloadOnSearch: false
       })
+      .when('/project/:project/browse/provisioned', {
```

Comment:
I think it would be good to match the API kind in the code even if we prefer to it as "Provisioned Service" in the UI. So I'd suggest `/browse/service-instances/` and `/browser/services-instances/:instance` with corresponding names to the controller and views.